public/index.html

```diff
@@ -42,31 +42,36 @@
       body {
         margin: 0;
         padding: 0;
-        font-family: sans-serif
+        font-family: Roboto,Open Sans,Droid,Ubuntu,sans-serif
       }
 
       .StationView h2 {
         text-transform: capitalize
       }
 
-      body {
-        font-family: Roboto,Open Sans,Droid,Ubuntu,sans-serif
+      .ContainerStationPills {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-end;
+        flex-direction: row;
+        margin-bottom: -5px;
       }
 
-      div.Pill {
+      .Pill {
         margin-left: 5px;
         font-weight: 500;
-        font-family: monospace
+        font-family: monospace !important;
```

Comment:
trading this `!important` for the extra specificity provided by `div.Pill` is really just a personal preference... I can easily be argued off of it haha.